src/main/resources/jenkins/branch/DescriptionColumn/column.jelly

```diff
@@ -0,0 +1,14 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core">
+  <td>
+    <j:set var="p" value="${col.getPropertyOf(job)}"/>
+    <j:choose>
+      <j:when test="${p != null and p.objectUrl != null}">
+        <a href="${p.objectUrl}">${col.description(p,job)}</a>
```

Comment:
Are `description`s (and `displayName`s) expected to be plain text or may they contain HTML? The assumption here is that they are plain text.